packages/expo-updates/e2e/fixtures/Updates.e2e.ts

```diff
@@ -818,7 +818,8 @@ describe('JS API tests', () => {
     jestExpect(didCheckAndDownloadHappenInParallel).toEqual('false');
   });
 
-  it('Receives expected events when update available on start', async () => {
+  // This test disabled, since these events are deprecated
```

Comment:
I think we should keep this enabled until we remove the events since there are still people relying upon them. Is the goal to remove the events?

Reply:
Yes that is the goal. They were marked deprecated in SDK 50, so technically people should not rely on them any more.

Reply:
Cool, I'll create a task to remove the events for 51: https://linear.app/expo/issue/ENG-11779/remove-deprecated-expo-updates-events-for-sdk-51